Framework/Algorithms/src/CalculatePlaczek.cpp

```diff
@@ -34,25 +34,41 @@ using Mantid::Kernel::Direction;
 using PhysicalConstants::BoltzmannConstant;
 using PhysicalConstants::E_mev_toNeutronWavenumberSq; // in [meV*Angstrom^2]
 
+struct summationTerms {
```

Comment:
You might as well use a std::pair<double, double> instead since your variables are named "first" and "second" anyway. Then you can do away with the struct entirely and the code will be more readable.

Reply:
@thomashampson Changes pushed according to your comments. Thanks a lot!